File: Documents/prog25/algorithms/05-array.py
# https://www.w3schools.com/python/python_arrays.asp
# python não suporta vetores, mas listas podem ser usadas

# declara uma lista
numbers = []

# repete 10 vezes
# pode utilizar apenas 1 número no "range", nesse caso
# será equivalente a variar i de 0 a 9 
# (para i=10 não entra no loop)
for i in range(10):
    # pede um número
    n = input("Digite um número: ")
    # guarda o número na lista
    numbers.append(n)
    
# percorre o vetor e mostra os números
for i in range(10):
    print(f"vetor[{i}] = {numbers[i]}")    

File: Documents/prog25/algorithms/05-array.c
// https://www.w3schools.com/c/c_arrays.php

#include <stdio.h>

int main()
{
    // declara um vetor para até 10 números
    int numbers[10];
    // repete 10 vezes
    for (int i=0; i <=9; i++) {
        // pede um número
        printf("Digite um número: ");
        int n;
        scanf("%d", &n);
        // guarda o número em uma posição de vetor
        numbers[i] = n;
    }
    
    // percorre o vetor e mostra os números
    for (int i=0; i <=9; i++) {
        printf("vetor[%d] = %d\n", i, numbers[i]);
    }
    return 0;
}